src/dsp_tools/commands/xmlupload/models/ontology_problem_models.py

```diff
@@ -113,3 +115,54 @@ def _format_prop(prop_tup: tuple[str, list[str], str]) -> str:
             return "The following resource(s) have invalid property type(s):\n\n" + medium_separator.join(problems)
         else:
             return None
+
+
+@dataclass
+class InvalidTextValueEncodings:
+    """
+    This class contains information about resources and the respective properties that have invalid text encodings.
+
+    An invalid encoding would be a property that specifies an xml encoding in the ontology,
```

Comment:
I know what you mean, but it's technically not correct: The onto cannot specify an xml encoding, but only the salsah-gui attributes "Richtext", "Textarea", "SimpleText"

Reply:
I know, I quite dislike the way we name this. Using encoding to specify this in the xml is quite weird as xml themselves are usually utf8 encoded.

Reply:
Me too, I dislike the distinction "xml" versus "utf8". Technically, that's an awful distinction.

Now, I think that the docstring is better 👍